_opensearch/query-dsl/term.md

```diff
@@ -25,6 +25,7 @@ You can use any of the term-level query types shown in the following table:
 `range` | Specifies a search within an upper and lower bound of values in a provided field.
 `prefix` | Specifies a search for terms that begin with a prefix.
 `exists` | Specifies a search for documents that contain a specific field.
+`fuzzy` | Specifies a [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance) that measures the number of one-character changes needed to change one term to another term. The fuzzy query creates sets of all possible variations of a term specified by the `value` field, and returns exact matches for terms that match the specified edit distance.    
```

Comment:
I don't think we should go into details here because we're not doing it for other query types.
```suggestion
`fuzzy` | Returns terms that are similar to the search term within the maximum allowed [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance). The Levenshtein distance measures the number of one-character changes needed to change one term to another term.
```

Reply:
ok thanks for fine-tuning the definition.